cukas.js

```diff
@@ -40,14 +40,15 @@ class CukasGame {
         if(playerId==(this.activePlayerId+1)%this.players.length) playersState = 2;
         //CukasPlayerPerspective.STATE_DEFENDING;
         //var Perspective=new CukasPlayerPerspective(CardSet.copy(his.players[playerId].hand), [...this.attack], [...this.defence], otherHands, Card.copy(this.trump), playersState);
+        //{hand,attack,defence,otherHands,trumpis,state}
```

Comment:
Šis komentārs nebūtu nepieciešams, ja sekojošā rinda tiktu uzskatāmi izvērsta pa vairākām rindiņām:
```
{
    a: 1,
    b: "abc"
}
```

Reply:
Done :)